src/IssueViz.Security/OpenInIDE_Hotspots/HotspotsList/OpenInIDEHotspotsControl.xaml

```diff
@@ -97,8 +108,19 @@
                 <DataGridTemplateColumn CanUserSort="True" SortMemberPath="Hotspot.Issue.RuleKey" Header="Code" Width="130" MinWidth="54">
                     <DataGridTemplateColumn.CellTemplate>
                         <DataTemplate>
-                            <TextBlock Text="{Binding Hotspot.Issue.RuleKey}" 
-                                       Style="{StaticResource CellTextBlockStyle}" />
+                            <TextBlock>
+                                <Hyperlink Style="{StaticResource RuleKeyLinkStyle}" 
+                                           Command="{Binding DataContext.NavigateToRuleDescriptionCommand, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType=DataGrid}}">
+                                
+                                    <Hyperlink.CommandParameter>
+                                        <MultiBinding Converter="{StaticResource NavigateToRuleDescriptionCommandConverter}">
+                                            <Binding Path="Hotspot.Issue.RuleKey" />
```

Comment:
@georgii-borovinskikh-sonarsource the XAML is exactly the same as in your PR, except that I removed the leading `.` from the binding paths (I was getting a warning that the path was invalid).

FYI as a test I removed the leading `.` from the corresponding binding paths local Hotspots window and binding still seemed to work i.e. rule navigation in local rules still worked. Weird.